XPlat/TensorFlow.Lite/build.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
```

Comment:
We don't want to duplicate the build script files in this dir, we can just use the one from the root and delete this.